compose.yaml

```
@@ -0,0 +1,96 @@
+      target: dev
+    env_file: [.env.local]
+    environment:
+      NODE_ENV: development
+      DATABASE_URL: postgres://${POSTGRES_USER:?}:${POSTGRES_PASSWORD:?}@postgres:5432/${POSTGRES_DB:?}
+      REDIS_URL: redis://redis:6379/0
+      SMTP_HOST: mailsink
+    ports: ["3000:3000"]
+    volumes:
+      - chat-attachments:/data/attachments
+    depends_on:
+      postgres: { condition: service_healthy }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)